src/Dfc.CourseDirectory.WebV2/Features/TLevels/ViewAndEditTLevel/ViewTLevel.cshtml

```diff
@@ -12,8 +12,8 @@
 
         <h1 class="govuk-heading-xl">Your T Level information</h1>
         <h2 class="govuk-heading-l">@Model.TLevelDefinitionName</h2>
-        <!-- Removed until we know where the link is going.
-    <p class="govuk-body">You can <a href="javascript:preventdefault();">view your published T Level (opens in new window or tab)</a> to see how it's viewed by learners.</p>-->
+        
+        <p class="govuk-body">You can <a href="https://nationalcareers.service.gov.uk/find-a-course/tdetails?tlevelId=@Model.TLevelId" target="_blank" class="govuk-link">view your published T Level</a> to see how it's viewed by learners.</p>
```

Comment:
Story is somehow ambiguous as to whether this link should open in a new window, so I've gone with should as per previous prototypes.